cmd/store/import.go

```diff
@@ -35,79 +36,157 @@ import (
 	"github.com/openfga/cli/internal/storetest"
 )
 
-func importStore(
+const (
+	progressBarWidth         = 40
+	progressBarSleepDelay    = 10 // time.Millisecond
+	progressBarThrottleValue = 65
+)
+
+func createStore(
 	clientConfig fga.ClientConfig,
-	fgaClient client.SdkClient,
 	storeData *storetest.StoreData,
 	format authorizationmodel.ModelFormat,
-	storeID string,
-	maxTuplesPerWrite int,
-	maxParallelRequests int,
 	fileName string,
 ) (*CreateStoreAndModelResponse, error) {
-	response := &CreateStoreAndModelResponse{}
+	storeDataName := storeData.Name
+	if storeDataName == "" {
+		storeDataName = strings.TrimSuffix(path.Base(fileName), ".fga.yaml")
+	}
 
-	if storeID == "" { //nolint:nestif
-		storeDataName := storeData.Name
-		if storeDataName == "" {
-			storeDataName = strings.TrimSuffix(path.Base(fileName), ".fga.yaml")
-		}
+	createStoreAndModelResponse, err := CreateStoreWithModel(
+		clientConfig,
+		storeDataName,
+		storeData.Model,
+		format,
+	)
+	if err != nil {
+		return nil, err
+	}
 
-		createStoreAndModelResponse, err := CreateStoreWithModel(
-			clientConfig,
-			storeDataName,
-			storeData.Model,
-			format,
-		)
-		if err != nil {
-			return nil, err
-		}
+	clientConfig.StoreID = createStoreAndModelResponse.Store.Id
```

Comment:
It looks like because we're passing `clientConfig` to this function by value rather than by reference the setting of `StoreID` is only happening in the scope of this function causing the tuples to not be written due to a `storeId is required and must be specified` error.

Unfortunately, because this isn't reported as an error by `ImportTuples` it doesn't get returned as an error

Reply:
This will be resolved in a4b7d05